SmartDeviceLink/SDLIAPSession.m

```diff
@@ -51,9 +68,17 @@ - (BOOL)start {
 
         strongSelf.streamDelegate.streamErrorHandler = [self streamErroredHandler];
         strongSelf.streamDelegate.streamOpenHandler = [self streamOpenedHandler];
-
-        [strongSelf startStream:weakSelf.easession.outputStream];
-        [strongSelf startStream:weakSelf.easession.inputStream];
+        if (!self.isDataSession){
+            [strongSelf startStream:weakSelf.easession.outputStream];
+            [strongSelf startStream:weakSelf.easession.inputStream];
+        }
+        else{
+            strongSelf.streamDelegate.streamHasSpaceHandler = [self streamHasSpaceHandler];
+            // Start I/O event loop processing events in iAP channel
+            _ioStreamThread = [[NSThread alloc] initWithTarget:self selector:@selector(accessoryEventLoop) object:nil];
```

Comment:
If it's not in `init` or `dealloc`, properties should be referenced via `self`, unless there's a really good reason.